test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -825,4 +876,145 @@ public static Gen<DecoratedKey> decoratedKeys(Gen<IPartitioner> partitionerGen,
     {
         return rs -> partitionerGen.generate(rs).decorateKey(keyGen.generate(rs));
     }
+
+    public static String insertCqlAllColumns(TableMetadata metadata)
```

Comment:
I prob should migrate all usage to the `ast.Statement`, this was kinda a limited hack and switching to the ast made things far better covered